### DISCUSSION TITLE
Auto cycle screens

### DISCUSSION THREAD

**Discussion by MyOwn2C** [BitMaker-hub/NerdMiner_v2#313] [Ideas]
How about a feature with option to enable auto cycling of the different screens?
- User can select which screens to be included in the cycle (Mining, Clock, World Stats, BTC price, etc)
- User can enter amount of time delay to cycle to next screen

**Comment by sany3001** [comment]
Like [carousel in Meshtastic project](https://meshtastic.org/docs/configuration/radio/display/).

Update: I am asking to have an option to choose also default screen without cycling.
